cmd/helm/completion.go

```diff
@@ -52,13 +56,14 @@ func newCompletionCmd(out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "completion SHELL",
-		Short: "Generate autocompletions script for the specified shell (bash or zsh)",
+		Short: "Generate autocompletions script for the specified shell (bash, zsh or fish)",
 		Long:  completionDesc,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runCompletion(out, cmd, args)
 		},
 		ValidArgs: shells,
 	}
+	cmd.PersistentFlags().BoolVar(&completionNoDesc, "no-descriptions", false, "disable completion description for shells that support it")
```

Comment:
Don't introduce this flag yet, until we know that Cobra will support disabling descriptions.  Or else we'll be stuck supporting this flag without being able to use Cobra for Fish completion.